src/components/MainView/main-view.jsx

```diff
@@ -112,39 +109,50 @@ export class MainView extends Component {
     this.props.setLoadingStatus(true);
     const validHooks = ['patient-view', 'order-select'];
```

Comment:
order-sign?

Reply:
I don't think we have good/working support for order-sign yet; so I'd prefer leave this to a separate PR, since there are some UX questions and it would be net-new functionality.